convert_util.c

```diff
@@ -162,6 +162,23 @@ _convert_write_tlv_error(uint8_t *buff, size_t buff_len,
 	return length;
 }
 
+static ssize_t
+_convert_write_tlv_extended_tcp_hdr(uint8_t *buff, size_t buff_len,
+                                    UNUSED struct convert_opts *opts)
+{
+	struct convert_extended_tcp_hdr *ext_tcp_hdr =
+		(struct convert_extended_tcp_hdr *)buff;
+	size_t length = CONVERT_ALIGN(
+		sizeof(*ext_tcp_hdr));
+
+	if (buff_len < length)
+		return -1;
+
+	/* TODO support actually announcing TCP options */
```

Comment:
Here you can go through the opts->options and write them in the TLV.

Reply:
Done